config/reports/links-between-worship-services-and-admin-units-report.js

```diff
@@ -82,7 +87,7 @@ export default {
       'adminUnit',
```

Comment:
This snippet is missing these two still, right?

```suggestion
      'worshipAdministrativeUnitRelationship',
      'worshipAdministrativeUnitRelationshipLabel'
      'adminUnit',
```

Reply:
Yes you're right, not sure how I missed that when I tested it